### PR TITLE
Update pihole-google.txt

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

File: pihole-google.txt [hunk 1]
```diff
@@ -7688,3 +7688,24 @@ www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
 yellowstone.jibemobile.com
+# More
+300zlf-dnsotls-ds.metric.gstatic.com
+cdn.fastclick.net
```
Comment (nickspaargaren) [inline pihole-google.txt:7693]:
```suggestion
```

File: pihole-google.txt [hunk 2]
```diff
@@ -7688,3 +7688,24 @@ www.tokyo.jibemobile.com
...
+e.crashlytics
+gstaticadssl.l.google.com
+KQu0cL-dnsotls-ds.metric.gstatic.com
+media.fastclick.net
```
Comment (nickspaargaren) [inline pihole-google.txt:7697]:
```suggestion
```

File: pihole-google.txt [hunk 3]
```diff
@@ -7688,3 +7688,24 @@ www.tokyo.jibemobile.com
...
+sa-in-f188.1e100.net
+tg-in-f188.1e100.net
+wa-in-f188.1e100.net
+wl-in-f188.1e100.net
+wq-in-f188.1e100.net
+www-google-analytics.l.google.com
+alt5-mtalk.google.com.access.network
+r2---sn-4g5edn6k.googlevideo.com.access.network
+r5---sn-25ge7nsl.googlevideo.com.access.network
+r5---sn-25glen7y.googlevideo.com.access.network
```
Comment (nickspaargaren) [inline pihole-google.txt:7711]:
```suggestion
```

File: pihole-google.txt [hunk 4]
```diff
@@ -7688,3 +7688,24 @@ www.tokyo.jibemobile.com
...
+optimizationguide-pa.googleapis.com
+passwordsleakcheck-pa.googleapis.com
+peoplestackwebexperiments-pa.clients6.google.com
+pf-in-f188.1e100.net
+sa-in-f188.1e100.net
+tg-in-f188.1e100.net
+wa-in-f188.1e100.net
+wl-in-f188.1e100.net
+wq-in-f188.1e100.net
+www-google-analytics.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7707]:
```suggestion
```

This one is already in the [regex](https://github.com/nickspaargaren/no-google#warning-important-regex-filters-not-included-in-blocklist).

File: pihole-google.txt [hunk 5]
```diff
@@ -7688,3 +7688,24 @@ www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
 yellowstone.jibemobile.com
+# More
+300zlf-dnsotls-ds.metric.gstatic.com
+cdn.fastclick.net
+e.crashlytics
```
Comment (nickspaargaren) [inline pihole-google.txt:7694]:
```suggestion
```

Invalid domain name

Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt (hunk 4 above)
Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt (hunk 3 above)
Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt (hunk 2 above)
Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt (hunk 1 above)
Comment (gallegonovato) [reply pihole-google.txt]: Done

File: pihole-google.txt [hunk 6]
```diff
@@ -7688,3 +7688,16 @@ www.tokyo.jibemobile.com
 www.viper.jibemobile.com
 www.watermelon.jibemobile.com
 yellowstone.jibemobile.com
+# More
+300zlf-dnsotls-ds.metric.gstatic.com
+gstaticadssl.l.google.com
```
Comment (nickspaargaren) [inline pihole-google.txt:7693]:
```suggestion
```
regex

**Pull request merged**